examples/secret_config.py

```diff
@@ -0,0 +1,6 @@
+config = dict(sqlalchemy_database_uri="postgresql://analysis_user:connectallthethings@35.196.105.34/postgres",
```

Comment:
i wonder if we want to keep this kind of thing out of git

Reply:
Yeah, good point. There's no need to have it in here or in the BaseConfig in the server.config file. I just pushed a couple of changes that defaults that variable to None and will get the value from config, just like deployment.